components/prism-nand2tetris-hdl.js

```diff
@@ -0,0 +1,8 @@
+Prism.languages['nand2tetris-hdl'] = {
+	'comment': /\/\/.*|\/\*[\s\S]*?(?:\*\/|$)/,
+	'keyword': /\b(?:CHIP|IN|OUT|PARTS|BUILTIN|CLOCKED)\b/,
+	'boolean': /\b(?:true|false)\b/,
+	'number': /\b\d+\b/,
+	'operator': /=/,
```

Comment:
There also seems to be a [range operator `..`](https://github.com/pmenon/nand-2-tetris/blob/master/projects/05/Memory.hdl#L35) (I guess).

Reply:
True, I accidentally put the single dot as punctuation, but it is rather just a double dot range operator here. `'operator': /=|\.\./` should do the trick, right?

Reply:
Yes, that should work.